### ISSUE TITLE
packing does not work properly, which makes the accept_loop shellcode fail.

### ISSUE THREAD

**Issue by kokjo** [Gallopsled/pwntools#415]
```
$ shellcraft i386.linux.acceptloop_ipv4 999
Traceback (most recent call last):
  File "/usr/local/bin/shellcraft", line 9, in <module>
    load_entry_point('pwntools==2.2.0', 'console_scripts', 'shellcraft')()
  File "/home/jonas/code/pwntools/pwnlib/commandline/shellcraft.py", line 239, in main
    code = func(*args.args)
  File "<string>", line 6, in acceptloop_ipv4
  File "/usr/local/lib/python2.7/dist-packages/mako/template.py", line 443, in render
    return runtime._render(self, self.callable_, args, data)
  File "/usr/local/lib/python2.7/dist-packages/mako/runtime.py", line 803, in _render
    **_kwargs_for_callable(callable_, data))
  File "/usr/local/lib/python2.7/dist-packages/mako/runtime.py", line 835, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/usr/local/lib/python2.7/dist-packages/mako/runtime.py", line 860, in _exec_template
    callable_(context, *args, **kwargs)
  File "/home/jonas/.pwntools-cache/mako/i386/linux/acceptloop_ipv4.asm.py", line 55, in render_body
    __M_writer(unicode(i386.linux.push('AF_INET | (%d << 16)' % htons(port))))
  File "<string>", line 6, in push
  File "/usr/local/lib/python2.7/dist-packages/mako/template.py", line 443, in render
    return runtime._render(self, self.callable_, args, data)
  File "/usr/local/lib/python2.7/dist-packages/mako/runtime.py", line 803, in _render
    **_kwargs_for_callable(callable_, data))
  File "/usr/local/lib/python2.7/dist-packages/mako/runtime.py", line 835, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/usr/local/lib/python2.7/dist-packages/mako/runtime.py", line 860, in _exec_template
    callable_(context, *args, **kwargs)
  File "/home/jonas/.pwntools-cache/mako/i386/linux/push.asm.py", line 33, in render_body
    __M_writer(unicode(i386.push(value)))
  File "<string>", line 6, in push
  File "/usr/local/lib/python2.7/dist-packages/mako/template.py", line 443, in render
    return runtime._render(self, self.callable_, args, data)
  File "/usr/local/lib/python2.7/dist-packages/mako/runtime.py", line 803, in _render
    **_kwargs_for_callable(callable_, data))
  File "/usr/local/lib/python2.7/dist-packages/mako/runtime.py", line 835, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/usr/local/lib/python2.7/dist-packages/mako/runtime.py", line 860, in _exec_template
    callable_(context, *args, **kwargs)
  File "/home/jonas/.pwntools-cache/mako/i386/push.asm.py", line 63, in render_body
    __M_writer(unicode(re.sub(r'^\s*/.*\n', '', i386.pushstr(packing.pack(value, 32, 'little', True), False), 1)))
  File "/home/jonas/code/pwntools/pwnlib/util/packing.py", line 121, in pack
    raise ValueError("pack(): number does not fit within word_size")
ValueError: pack(): number does not fit within word_size
$ 
```

**Comment by IdolfHatler** [comment]
Can easily be fixed if somebody were to fix #338 first.

**Comment by zachriggle** [comment]
This appears to be fixed in `master`.

```
$ shellcraft i386.linux.acceptloop_ipv4 999
6a66586a015b9952536a0289e1cd80526801010101813424030102e689e16a10515089e189c643b066cd80b066b304cd80525689e1b066b305cd8089c5b002cd809785ff89eb0f44de6a0658cd8085ff75df
```
